### PR TITLE
Modal global styles

### PR DESCRIPTION
**Pull request by nikku**
* Modal styles are moved to global stylesheet.
* Modal structure is aligned with bootstrap@4
* Add some a11y hints to improve accessibility
* Some superfluous style declarations got removed

---

Rationale: We did the same thing for buttons already. I believe it benefits us, if we do not invent our own CSS framework for things that are properly taken care of by popular frameworks.

### CODE REVIEW

File: client/src/app/primitives/modal/Modal.js [hunk 1]
```diff
@@ -62,10 +60,12 @@ export default class Modal extends PureComponent {
     } = this.props;
 
     return ReactDOM.createPortal(
-      <div className={ css.ModalOverlay }>
-        <div className={ classNames(css.ModalContainer, className) } ref={ this.modalRef }>
-          { children }
-          { onClose && (<Close onClick={ this.close } />) }
+      <div className="modal" tabIndex="-1" role="dialog">
```
Comment (barmac) [inline client/src/app/primitives/modal/Modal.js]: Why do we need `tabIndex` here? AFAIK `div`s are not focusable per default.

Comment (nikku) [reply client/src/app/primitives/modal/Modal.js]: This should answer it: https://stackoverflow.com/questions/32911355/whats-the-tabindex-1-in-bootstrap-for

TLDR: With `tabIndex=-1` we can focus the element programmatically. We do not do that yet but _could_ do it. It also helps a11y technologies.
